common/src/main/resources/common/setup-users.feature

```diff
@@ -78,7 +78,7 @@ Feature: prepare data for api test
     Given path '/perms/permissions'
     And header x-okapi-tenant = testTenant
     And param length = 1000
-    And param query = '(subPermissions="" NOT subPermissions ==/respectAccents []) and (cql.allRecords=1 NOT childOf <>/respectAccents [])'
+    And param query = 'subPermissions="" and (cql.allRecords=1 NOT childOf <>/respectAccents [])'
```

Comment:
```suggestion
    And param query = 'childOf == "[]"'
```

RMB always creates an emtpy array if the client doesn't provide a `childOf` property. Therefore we don't need the `NOT` construct to also match a missing `childOf` property.

`subPermissions=""` is the same as `cql.allRecords=1` because RMB always creates an empty array if the client doesn't provide a `subPermissions` property.

`/respectAccents` is valid CQL syntax but ignored by RMB. To influence the matching it must be specified in schema.json.

The JSON array `[]` brackets are always without accents, therefore `/respectAccents` is not needed.